### ISSUE TITLE
nPose Core 2.0 for OSGrid creates extra seats..

### ISSUE THREAD

**Issue by MuffeySummers** [HowardBaxton/nPose#89]
I believe I have a bug with nPose Core 2.0 for OSGrid.

I created a chair, with a DEFAULT anim set for 3 seats -

ANIM|chair01_female|<0.000121, -0.168373, 0.177887>|<0.000000, 0.000000, 0.000000>|
ANIM|Sit 160M|<0.000009, -0.012306, 0.029587>|<-0.000002, 0.002431, -89.958702>|
ANIM|chair02_female|<0.000121, -0.168373, 0.177887>|<0.000000, 0.000000, 0.000000>|

I've added some SCHMO poses to change those seats up, and they work fine.

I added some ANIM poses for a 2 person cuddle, but if I -

1) sit on chair
2) move to seat 3
3) choose 'Cuddle' - which is a 2 person ANIM 

It doesn't move me, and it *creates* 2 extra seats. At that point, all seat movement / changing is broken and it has to have a reset scripts done on it.

While I understand I can change my ANIM cuddle to be a SCHMOE for seats 1~2, I would *expect* the behaviour would be to kick off seat 3 and change seats 1 and 2.

![npose-issue](https://cloud.githubusercontent.com/assets/25622623/22713207/7c485934-ed55-11e6-9d4e-325510b1b4e0.png)

**Comment by LeonaMorro** [comment]
@HowardBaxton : is this still an unsolved issue?

**Comment by HowardBaxton** [comment]
Yes I believe it has.  This was an issue where a variable was defined outside of the for( ; index<stop; ++index).  OSGrid does not work as expected in this situation.  It has to be written like this: for(index=0; index<stop; ++index).  After this all worked as expected.  Just a difference between SL and OSGrid.
